### ISSUE TITLE
Populate Related Posts

### ISSUE THREAD

**Issue by jcahill** [FraserThompson/PhilosophersMeme#1]
> @74yearoldsocialist: Hey, when did the "related posts" feature break on the website?

looks like 01c4267b7 introduces a bad `page.title != page.title` check

```
 		  	{% for tag in page.tags %}
  			  	{% for post in site.tags[tag] %}
 -				  	{% if post.title != null and post.title != page.title and post.status != "draft" %}
 -				  	 {% capture related_posts_w %}{{related_posts_w}}{{post.url}}^{{post.title}}^{{post.image}}^{{post.author.display_name}}^{{post.date}}|{% endcapture %}
 +				  	{% if page.title != null and page.title != page.title and page.status != "draft" %}
 +				  	 {% capture related_posts_w %}{{related_posts_w}}{{page.url}}^{{page.title}}^{{page.image}}^{{page.author.display_name}}^{{page.date}}|{% endcapture %}
  				  	{% endif %}
  			  	{% endfor %}
  		  	{% endfor %}
```

**Comment by FraserThompson** [comment]
Thanks. Should be checking against post.title to ensure the post itself doesn't appear in related posts. Fixed in 16b9aa9cbc1b9476bcd70164acd0401234dc09b2

**Comment by FraserThompson** [comment]
Scrap that, it's something else. Still broken. Will fix.

**Comment by FraserThompson** [comment]
Okay, properly fixed now. Looks like it was a bad search/replace or something which turned all the occurrences of post into page. A bit concerning that it remained broken for so long, makes me think we might need some sort of testing process.

Fixed in 158e6950f626a27459737299ff7b6ba44f8235e8
